app/components/todo.hbs

```diff
@@ -1,11 +1,29 @@
-<form onsubmit={{this.sumbit @model}}>  
```

Comment:
There are two schools of thought regarding forms implementation:

1.  Use the form submit event. In this case, you don't need to memoise input value into `this.text` because you will receive the form data in the submit event object.
2. Manipulate fields and buttons directly. In this case, you don't need the `<form>` element at all. You can still use `<form>` for a11y, but don't put events on it. Events go on individual fields and buttons.

I used the latter approach in this PR.